ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -52,12 +52,12 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )edge-ci-index,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
can we fine-tune on what cases we're running it?
because as-is it will run even on docs changes

Reply:
to clarify, how central is this flow? should it run more frequently than capi/regular-ztp?

Reply:
I think it should run the same as the basic ZTP job.
I'll adjust it